.changeset/hip-cherries-behave.md

```diff
@@ -0,0 +1,7 @@
+---
+"astro": patch
+---
+
+Improves Node.js streaming performance
+
+This uses an AsyncIterable instead of a ReadableStream to do streaming in Node.js. This is a non-standard enhancement by Node, so this is done only in that environment.
```

Comment:
What's the implication of this being "a non-standard enhancement by Node" and why does the reader care about this? 

"Non-standard" doesn't always imply something positive, (is this a hack? are we jumping on to something experimental?) and in fact might make people worry/wonder. So this feels maybe a little too jargon-y when I'm assuming that this could actually be framed as something cool and helpful!

Just a thought!

Reply:
It just means that it won't be run non-Node environments, for ex. in Cloudflare. Those will take the slower path. 

Yeah it can be tricky, I'm not always sure who the target of these changesets are. A lot of changes are implementation details that most won't understand or care about, but some will. And they can be useful for understanding the change on the part of maintainers. But we don't usually do that.

Thinking... do you have a suggested way to rephrase this?